### PR TITLE
resolve audit failure from cve 1753 (trim-newlines) & update web template

### PR DESCRIPTION
**Pull request by aruniverse**
- Fixes audit failure coming from [cve-1753](https://www.npmjs.com/advisories/1753)
  - Drop unnecessary dep on node-sass which had a sub dep on trim-newlines

- Bump react-scripts to 3.4.13, to pick up fixes for [cve-1755](https://www.npmjs.com/advisories/1755), see [pr](https://github.com/imodeljs/create-react-app/pull/44)

- Add script to run audit against npm registry

- Add assets to template for favicon, manifest, etc

### CODE REVIEW

File: common/scripts/audit.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    logBuildWarning("Rush audit failed. This may be caused by a problem with the npm audit server.");
+  }
+
+  // A list of temporary advisories excluded from the High and Critical list.
+  // Warning this should only be used as a temporary measure to avoid build failures
+  // for development dependencies only.
+  // All security issues should be addressed asap.
+  const excludedAdvisories = [
+    1674, // https://npmjs.com/advisories/1674
+    1700, // https://npmjs.com/advisories/1700
```
Comment (calebmshafer) [inline common/scripts/audit.js:41]: @aruniverse are these the ones you're hitting in this pipeline?

Comment (aruniverse) [reply common/scripts/audit.js:41]: i copied these directly from the core repo, i assumed id hit the same ones core does for the most part

Comment (aruniverse) [reply common/scripts/audit.js:41]: ill double check to be safe

**Review by StefanRetief — APPROVED** [review]

File: common/scripts/audit.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  }
+
+  // For some reason yarn audit can return the json without the vulnerabilities
+  if (undefined === jsonOut.metadata.vulnerabilities || shouldFailBuild)
+    failBuild();
+
+  process.exit();
+})();
+
+function runYarnAuditAsync(cwd) {
```
Comment (kckst8) [inline common/scripts/audit.js:86]: doesn't this add a dep on yarn? will this work if you uninstall yarn globally

Comment (aruniverse) [reply common/scripts/audit.js:86]: switched to using the pnpm  one now that we got that other pr through, audit looks good now.

File: packages/modules/cra-template-web-viewer/template.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
       "@bentley/presentation-frontend": "^2.14.0",
       "@bentley/product-settings-client": "^2.14.0",
       "@bentley/rbac-client": "^2.14.0",
-      "@bentley/react-scripts": "^3.4.9",
+      "@bentley/react-scripts": "^3.4.13",
```
[the remark this answers is not in the document]
Comment (aruniverse) [reply packages/modules/cra-template-web-viewer/template.json:21]: bumped to stay consistent across monorepo.

**Review by kckst8 — APPROVED** [review]
works well on my end

**Pull request merged**